src/test/java/core/basesyntax/sevice/ConvertReadedDataServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.sevice;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ConvertReadedDataServiceImplTest {
+    private static ConvertReadedDataService convertReadedDataService;
+    private static List<String> inputDataList;
+    private static List<FruitTransaction> expectedList;
+
+    @BeforeClass
+    public static void beforeClass() {
+        convertReadedDataService = new ConvertReadedDataServiceImpl();
+        inputDataList = new ArrayList<>();
+        inputDataList.add("type,fruit,quantity");
+        inputDataList.add("s,apple,13");
+        inputDataList.add("s,banana,50");
+        expectedList = new ArrayList<>();
+        expectedList.add(new FruitTransaction("s", "apple", 13));
+        expectedList.add(new FruitTransaction("s", "banana", 50));
+    }
+
+    @Test
+    public void convertDataFromFile_Ok() {

```

Comment:
Rename each test method according to the rule from the readme, where the first letter of each part of the method name is lowercase
```suggestion
    public void convertDataFromFile_valideData_ok() {
```
<img width="857" alt="Screenshot 2022-02-23 at 13 06 19" src="https://user-images.githubusercontent.com/29378916/155313363-ab665ea2-fefe-4743-b90e-9fa67d7df0b3.png">